packages/ez-react/tests/common.ts

```diff
@@ -16,3 +17,14 @@ export const baseChartProps: ChartProps = {
   scopedSlots: {},
   isWrapped: false,
 };
+
+export const renderSVG = (
+  ui: React.ReactElement,
+  options?: Omit<RenderOptions, 'queries'>
+): RenderResult => {
+  const container = document.createElementNS(
+    'http://www.w3.org/2000/svg',
+    'svg'
+  );
+  return render(ui, { ...options, container });
+};
```

Comment:
```suggestion

export const renderSVG = (
  ui: React.ReactElement,
  options?: Omit<RenderOptions, 'queries'>
): RenderResult => {
  const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
  return render(ui, {
    ...options,
    container: svg as Element as HTMLElement,
  });
};
```

Reply:
@G0maa This should get rid of the TS error.

Reply:
Done! @marrouchi